Database loading checks

OscarVsp opened the pull request:
Added check and error msg for database loading to catch case where the credential is not correct, or the spreadheet/worksheet is not found.

Need testing (don't have proper environnement for now)

LucasPlacentino committed:
Update README.md

LucasPlacentino committed:
add memory limit to the container

OscarVsp committed:
Merge pull request #23 from bepolytech/main

update dev from main

OscarVsp committed:
add missing information about admin command

OscarVsp committed:
change disnake version from 2.6 to 2.7

OscarVsp committed:
removed frozenlist since it had compiler issu with python 3.11

OscarVsp committed:
added the feedback command information

OscarVsp committed:
Revert "add missing information about admin command"

This reverts commit d27108659aebeac3e3fed7d2576c8058b733ae25.

OscarVsp committed:
Revert "added the feedback command information"

This reverts commit 80e1829e05d6e2055d0425a3549834af418376ca.

OscarVsp committed:
added missing information about discord commands

OscarVsp committed:
added member count and percent

OscarVsp committed:
added member count to server info and /stats command

OscarVsp committed:
added /stats admin command info

OscarVsp committed:
change disnake to 2.8

OscarVsp committed:
change black type to python3

OscarVsp committed:
added way to send error log message without cmd context

OscarVsp committed:
added check and error msg for database loading

dependabot[bot] committed:
Bump aiohttp from 3.8.3 to 3.9.0

Bumps [aiohttp](https://github.com/aio-libs/aiohttp) from 3.8.3 to 3.9.0.
- [Release notes](https://github.com/aio-libs/aiohttp/releases)
- [Changelog](https://github.com/aio-libs/aiohttp/blob/master/CHANGES.rst)
- [Commits](https://github.com/aio-libs/aiohttp/compare/v3.8.3...v3.9.0)

---
updated-dependencies:
- dependency-name: aiohttp
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

LucasPlacentino committed:
sync fix to dev - MergePR #34 from bepolytech/main

sync main to dev - fix docker build

LucasPlacentino commented:
imo we should have a proper testing environment: stage server, test deployments, dummy data, and would also love to have automated real-world testing

LucasPlacentino committed:
update python image to 3.11.6-alpine- Update Dockerfile

LucasPlacentino committed:
bump asyncpg, frozenlist, multidict, yarl to latest- Update requirements.txt

LucasPlacentino committed:
bump async-timeout to latest - Update requirements.txt

OscarVsp committed:
Merge remote-tracking branch 'origin/dependabot/pip/aiohttp-3.9.0' into 19-update-to-disnake-270

LucasPlacentino committed:
fix: filter names to remove unwanted characters

only keep ASCII letters and spaces

LucasPlacentino committed:
rename user choice during update_all_guilds

during an update_all_guilds command, only updates the user's name if both the guild has rename enabled AND the admin set force rename to true at the command

LucasPlacentino committed:
add option for admin to choose to force rename during `/update` to all servers

LucasPlacentino committed:
Merge branch '19-update-to-disnake-270' into update-to-python-3-11-6-alpine

LucasPlacentino committed:
Merge pull request #37 from bepolytech/update-to-python-3-11-6-alpine

Update Python to 3.11.6 and upgrade dependencies, also fixes docker build

LucasPlacentino committed:
resolve merge conflict into bepolytech/dev

LucasPlacentino committed:
Merge branch 'dev' into update-option-rename

LucasPlacentino committed:
Merge pull request #44 from bepolytech/update-option-rename

fix #36 : rename user choice during update_all_guilds for admins' `/update`

LucasPlacentino committed:
Merge pull request #43 from bepolytech/fix-numbers-in-names

fix #42 : filter names to remove unwanted characters

LucasPlacentino committed:
fix typos

LucasPlacentino committed:
fix typo

LucasPlacentino committed:
Merge pull request #45 from bepolytech/19-update-to-disnake-270

into dev : 19 update to disnake 270

LucasPlacentino committed:
Update README.md

add info to `/update` command docs (force rename or not)

OscarVsp committed:
Better auto complete match by lowering the text

OscarVsp committed:
Better exception handler and logs

OscarVsp committed:
better logs
